### PR TITLE
Added test for volunteer flow through application.

### PR DESCRIPTION
**Pull request by BrendanBeachBCJ**
# Description

Added test that ensures the flow through the application for volunteers works as expected (bambora payment is skipped).

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Running test manually.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: *SPDBCSC-457*

### CODE REVIEW

File: src/eCRC_qaautomation/automationtest/smoketests/success_paymentcancel_spec.js [hunk 1]
```diff
@@ -1,191 +1,256 @@
...
 
-var testInput = require('../../input/success');
+var testInput = require("../../input/success");
 
-describe('success', function(){
+describe("success", function() {
+  beforeEach(function() {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000;
+  });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/smoketests/success_paymentcancel_spec.js]: are we not allowed to use arrow functions here?
```suggestion
describe("success", () => {
  beforeEach(() => {
    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
    jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000;
  });
```

File: src/eCRC_qaautomation/automationtest/smoketests/success_paymentnoresponse_spec.js [hunk 2]
```diff
@@ -1,195 +1,252 @@
...
 
-var testInput = require('../../input/success');
+var testInput = require("../../input/success");
 
-describe('success', function(){
+describe("success", function() {
+  beforeEach(function() {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
+  });
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/smoketests/success_paymentnoresponse_spec.js]: same question here and everywhere else applicable

File: src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+var bcscConsentPage = require("../../pageobjectfactory/bcscconsentpage");
+
+var consentPage = require("../../pageobjectfactory/consentpage.js");
+
+var applicationFormPage = require("../../pageobjectfactory/applicationformpage");
+
+var paymentPage = require("../../pageobjectfactory/paymentpage");
+
+var informationReviewPage = require("../../pageobjectfactory/informationreviewpage");
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js]: can we use `const` and `let`?
also can we use `import` over the old school `require`?

File: src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+var testInput = require("../../input/success");
```
Comment (sdevalapurkar-bcgov) [inline src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js]: ideally no `var`

File: src/eCRC_qaautomation/automationtest/smoketests/success_paymentnoresponse_spec.js (hunk 2 above)
Comment (BrendanBeachBCJ) [reply src/eCRC_qaautomation/automationtest/smoketests/success_paymentnoresponse_spec.js]: No reason not to use arrow notation I think. I'll switch it over.

File: src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js (hunk 3 above)
Comment (BrendanBeachBCJ) [reply src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js]: I'll update this as I go along. Will fix in this file right now.

File: src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js (hunk 4 above)
Comment (BrendanBeachBCJ) [reply src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js]: Same as above

**Pull request merged**